UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Helpers/CommonHelper.cs

```diff
@@ -12,5 +13,11 @@ public static IEnumerable<List<T>> SplitList<T>(List<T> masterList, int nSize =
                 yield return masterList.GetRange(i, Math.Min(nSize, masterList.Count - i));
             }
         }
+
+        public static int GetCurrentWeekNumber(DateTime date)
```

Comment:
Can we get some unit tests for this method please.

e.g.  2021/01/01 = 53, 2021/01/07=1, 2021/07/01=26?

Reply:
Resolved